apkleaks/apkleaks.py

```diff
@@ -75,6 +77,7 @@ def integrity(self):
 				self.dependencies()
 			else:
 				sys.exit(util.writeln("\n** Aborted.", col.FAIL))
+
```

Comment:
Remove this trailing newline.

Reply:
I actually added that one on purpose, so the code would be easier to read.